diff_drive_controller/CMakeLists.txt

```diff
@@ -15,7 +15,11 @@ catkin_package(
 )
 
 include_directories(
-  include ${catkin_INCLUDE_DIRS}
+  include
+)
+
+include_directories(SYSTEM
+ ${catkin_INCLUDE_DIRS}
```

Comment:
pedantic: This should be indented by two spaces.

Reply:
fixed and force pushed.